homeassistant/components/media_player/const.py

```diff
@@ -199,6 +199,7 @@ class MediaPlayerEntityFeature(IntFlag):
     BROWSE_MEDIA = 131072
     REPEAT_SET = 262144
     GROUPING = 524288
+    MEDIA_ENQUEUE = 2097152
```

Comment:
Should this be 1048576 (2^20)?

Reply:
That value is indeed reserved in https://github.com/home-assistant/core/pull/95906